### PR TITLE
RATYK-19: Apply black, isort

### PR DESCRIPTION
**Pull request by danipran**

### CODE REVIEW

File: munigeo/importer/manchester.py [hunk 1]
```diff
@@ -153,7 +162,7 @@ def import_pois_from_citadel(self):
             self._import_citadel(muni, d)
 
     def import_pois(self):
-        requests_cache.install_cache('geo_import_man')
+        requests_cache.install_cache("geo_import_man")
         self.logger.info("Importing POIs from Citadel")
         self.import_pois_from_citadel()
         # self.logger.info("Importing POIs from CSV")
```
Comment (nicobav) [inline munigeo/importer/manchester.py:168]: totally not in scope but we could remove this and below as well

File: munigeo/importer/athens.py [hunk 2]
```diff
@@ -1,84 +1,88 @@
...
 import csv
-#import unicodecsv
-import requests
-import requests_cache
 import io
 import json
+import os
 
-from django.conf import settings
+# import unicodecsv
```
[the remark this answers is not in the document]
Comment (danipran) [reply munigeo/importer/athens.py:7]: Now that I take a closer look at this (and the other examples), it seems there's a lot of unused imports as well. Probably will address these later, in another PR 😬

File: munigeo/importer/manchester.py [hunk 3]
```diff
@@ -1,72 +1,76 @@
...
 import csv
-#import unicodecsv
-import requests
-import requests_cache
 import io
 import json
+import os
 
-from django.conf import settings
+# import unicodecsv
```
[the remark this answers is not in the document]
Comment (danipran) [reply munigeo/importer/manchester.py:7]: Will address this later: https://github.com/City-of-Helsinki/django-munigeo/pull/63#discussion_r1469392714

Also worth mentioning, this (commented-out) import is actually used in the code, but the code itself doesn't call the method that uses the import (other than a commented-out line), so, uhh... yeah..

File: munigeo/importer/manchester.py (hunk 1 above)
Comment (danipran) [reply munigeo/importer/manchester.py:168]: Not addressing this yet. The manchester importer seems to need some work... or just straight-up deletion ¯\_(ツ)_/¯

**Pull request merged**